src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -579,4 +560,30 @@ class JavaMigrations {
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "14.ea.6-open", _))
   }
 
+	@ChangeSet(order = "182", id = "182-add_graalvm_19_2_0", author = "eddumelendez")
```

Comment:
Please fix the indention of the added code, so it uses the same code style as other parts in the file. Thanks in advance!

Reply:
I hate to do it, but I might be adding [scalafmt](https://scalameta.org/scalafmt/) to this project soon.

Reply:
I have applied the right format